pilot/pkg/model/context_test.go

```diff
@@ -395,23 +394,6 @@ func TestParseMetadata(t *testing.T) {
 	}
 }
 
-func mapToStruct(msg map[string]any) (*structpb.Struct, error) {
```

Comment:
Why move this function

Reply:
Constructing the `structpb.Struct` literal is too cumbersome, so I found this utility function. However, it was located in different packages (`model` vs `model_test`). Considering this is a rather practical utility function, I moved it to where it should belong.

Reply:
But we donot this function in any other place

Reply:
The current issue is that it needs to be used in both `model` and `model_test`, and either redefining it or exposing such a seemingly generic utility function in `model` package isn't a good practice. Additionally, I really feel that this utility function can be used in many places.

Reply:
ACK, i see it is used in test